integrations/brightcrowd/nango.yaml

```diff
@@ -0,0 +1,249 @@
+integrations:
+    brightcrowd:
+        syncs:
+            books:
+                runs: every day
+                description: |
+                    Fetches a list of all books in an account from Brightcrowd.
+                output: Book
+                track_deletes: true
+                sync_type: full
+                endpoint:
+                    method: GET
+                    path: /books
+                    group: Books
+                scopes: bcb.partner/book.read
+            booksById:
```

Comment:
by convention established by this repo let's make words `-` separated.

`books-by-id`

Same for the others